geniza/corpus/admin.py

```diff
@@ -207,47 +215,45 @@ def tabulate_queryset(self, queryset):
             all_textblocks = doc.textblock_set.all()
             all_footnotes = doc.footnotes.all()
 
-            row = {
+            initial_entry = doc.log_entries.first()
+            latest_revision = doc.log_entries.last()
+
+            row = self.DocumentRow(**{
                 'pgpid': doc.id,
-                # 'url': # site link
+                'url': absolutize_url(doc.get_absolute_url()),
                 'iiif_urls': ';'.join([fragment.iiif_url for fragment in all_fragments]),
-                'fragment_url': ';'.join([fragment.url for fragment in all_fragments]),
+                'fragment_urls': ';'.join([fragment.url for fragment in all_fragments]),
                 'shelfmark': doc.shelfmark,
                 'multifragment': ';'.join([tb.multifragment for tb in all_textblocks]),
                 'side': ';'.join([tb.side for tb in all_textblocks]),
                 'extent_label': ';'.join([tb.extent_label for tb in all_textblocks]),
                 'type': doc.doctype,
                 'tags': doc.all_tags(),
                 'description': doc.description,
-                # 'footnotes': ';'.join([str(fn) for fn in all_footnotes]),
+                'footnotes': ';'.join([str(fn) for fn in all_footnotes]),
                 'shelfmarks_historic': ';'.join([fragment.old_shelfmarks for fragment in all_fragments]),
                 'languages': doc.all_languages(),
                 'languages_probable': doc.all_probable_languages(),
                 'language_note': doc.language_note,
                 'notes': doc.notes,
                 'needs_review': doc.needs_review,
-                # 'url_admin': '',#reverse('admin:corpus_document_change', args=[doc.id]),
-                # input_by*
-                # date_entered (first log entry date)
-                'last_modified': doc.last_modified,
+                'url_admin': absolutize_url(reverse('admin:corpus_document_change', args=[doc.id])),
+                'initial_entry': f"{initial_entry.action_time}, {initial_entry.user.get_full_name() or initial_entry.user.get_username()}",
+                'latest_revision': f"{latest_revision.action_time}, {latest_revision.user.get_full_name() or latest_revision.user.get_username()}",
```

Comment:
And I'm indifferent. Who wants to be decisive about this? 😄

Reply:
I've thought about it more, and I would like to see the separate fields that Marina and I agreed on. I think that will be easier to work with. So that means initial entry date, last modified, and input by (unique list of all data editors).